certora/specs/SocialRecoveryModule.spec

```
@@ -302,3 +302,49 @@ rule disabledRecoveryModuleResultsInFinalizationRevert(env e) {
             safeContract.getOwners().length == 1 &&
             currentThreshold == safeContract.getThreshold());
 }
+
+// Recovery can be cancelled
+rule cancelRecovery() {
+    env e;
+    require e.msg.sender == safeContract;
+    require e.msg.value == 0;
+
+    // A recovery request must be initiated.
+    require currentContract.recoveryRequests[safeContract].executeAfter > 0;
+    require currentContract.walletsNonces[safeContract] > 0;
+
+    currentContract.cancelRecovery@withrevert(e, safeContract);
+    assert !lastReverted;
+}
+
+// Cancelling recovery for a wallet does not affect other wallets
+rule cancelRecoveryDoesNotAffectOtherWallet() {
+    env e;
+    address otherWallet;
```

```suggestion
rule cancelRecoveryDoesNotAffectOtherWallet(env e, address otherWallet) {
```

We can add the variables as parameters.

---

I also like parameterizing the rule with variables that are "important". But it is just a style thing :stuck_out_tongue: